src/test/java/io/strimzi/kafka/bridge/http/HttpBridgeTest.java

```diff
@@ -769,6 +769,144 @@ public void receiveSimpleMessageFromPartitionAndOffset(TestContext context) {
+                    String consumerInstanceId = bridgeResponse.getString("instance_id");
+                    String consumerBaseUri = bridgeResponse.getString("base_uri");
+                    context.assertEquals(name, consumerInstanceId);
+                    context.assertEquals(baseUri, consumerBaseUri);
+                    creationAsync.complete();
+                });
+
+        creationAsync.await();
+
+        // subscribe to a topic
+        Async subscriberAsync = context.async();
+
+        JsonObject subJson = new JsonObject();
+        subJson.put("topic", topic);
+
+        client.post(BRIDGE_PORT, BRIDGE_HOST, baseUri + "/subscription")
+                .putHeader("Content-length", String.valueOf(subJson.toBuffer().length()))
+                .as(BodyCodec.jsonObject())
+                .sendJsonObject(subJson, ar -> {
+                    context.assertTrue(ar.succeeded());
+                    context.assertEquals(204, ar.result().statusCode());
+                    subscriberAsync.complete();
+                });
+
+        subscriberAsync.await();
+
+        // consume records
+        Async consumeAsync = context.async();
+
+        client.get(BRIDGE_PORT, BRIDGE_HOST, baseUri + "/records")
+                .putHeader("timeout", String.valueOf(1000))
+                .as(BodyCodec.jsonArray())
+                .send(ar -> {
+                    context.assertTrue(ar.succeeded());
+
+                    HttpResponse<JsonArray> response = ar.result();
+                    JsonObject jsonResponse = response.body().getJsonObject(0);
+
+                    String kafkaTopic = jsonResponse.getString("topic");
+                    int kafkaPartition = jsonResponse.getInteger("partition");
+                    String key = jsonResponse.getString("key");
+                    String value = jsonResponse.getString("value");
+                    long offset = jsonResponse.getLong("offset");
+
+                    context.assertEquals(topic, kafkaTopic);
+                    context.assertEquals(sentBody, value);
+                    context.assertEquals(0L, offset);
+                    context.assertNotNull(kafkaPartition);
+                    context.assertNull(key);
+
+                    consumeAsync.complete();
+                });
+
+        consumeAsync.await();
+
+        Async commitAsync = context.async();
+
+        JsonArray offsets = new JsonArray();
+        json = new JsonObject();
+        json.put("topic", "commitOffset");
+        json.put("partition", 0);
+        json.put("offset", 1);
+        offsets.add(json);
+
+        JsonObject root = new JsonObject();
+        root.put("offsets", offsets);
+
+        client.post(BRIDGE_PORT, BRIDGE_HOST, baseUri + "/offsets")
+                .putHeader("Content-length", String.valueOf(root.toBuffer().length()))
+                .as(BodyCodec.jsonObject())
+                .sendJsonObject(root, ar -> {
+                    context.assertTrue(ar.succeeded());
+
```

Comment:
Too many empty lines

Reply:
Yeah I have another typo to fix, I will do directly on the master ;)